modules/apps/foundation/frontend-theme/theme3/theme-3-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,221 @@
+            background-attachment: fixed;
+            background-position: center;
+            background-repeat: no-repeat;
+            background-size: cover;
+        }
+            #navigation
+            {
+               background-color: #2D464C;
+               border: 20px;
+               border-color: #2D464C;
+               width: 100%;
+               height: 60px;
+               position: absolute;
+               bottom: 2140px;
+               z-index: 1;
+               display: block;
+
+               .active{
+                background-color: rgb(10, 27, 32);
+                    > a {
+                        color: #FFF;
+                    }
+            }        
+    
+            .dropdown-menu li {
+                background: rgb(25, 55, 70);
+                border-width: 0;
+                margin: 0;
+                position: relative;
+    
+                &:after, &:before {
+                    border-width: 0;
+                }
+    
+                a {
+                    color: #FFF;
+    
+                    &:hover {
+                        background: #000 none repeat scroll 0 0;
+                    }
+                }
+            }
+    
+            li {
+                margin: 0 4px;
+                padding: 15px 10px;
+                position: absolute;
+    
+                &:hover {
+                    background-color: #12272e;
+    
+                    .child-menu {
+                        display: block;
+                        margin-top: 0;
+                    }
+    
+                    a {
+                        color: #FFF;
+                        text-decoration: none;
+                    }
+                }
+            }
+    
+            ul {
+                li {
+                    &:hover {
+                    background-color:rgb(18, 39, 46);
+                        .child-menu {
+                            display: block;
+                            padding: 0;
+                            li {
+                                background-color: $primary-color;
+                                &:hover {
+                                    background-color:rgb(18, 39, 46);
+                                }
+                            }
+                            a {
+                                color: #FFF;
+                                margin: -10px;
+                                text-decoration: none;
+                            }
+                        }
+                    }
+                }
+            }
+    
+            > ul > li {
+                display: inline-block;
+                font-size: 1.2em;
+                position: relative;
+    
+                a {
+                    color: #FFF;
+                    padding: 15px 10px;
+                }
+            }
+         }
+
+        #content {
+            display: block;
+            font-family: Lato, Times, sans-serif;
```

Comment:
This is defined and used in other places.  Can we put this on a more global CSS class?